packages/react/src/components/List/Lists.tsx

```diff
@@ -1,66 +1,47 @@
 import { Slot } from '@radix-ui/react-slot';
 import cl from 'clsx/lite';
-import type { HTMLAttributes, OlHTMLAttributes } from 'react';
+import type { OlHTMLAttributes } from 'react';
 import { forwardRef, useContext } from 'react';
 
-import { Paragraph } from '../Typography';
-
 import { ListContext } from './ListRoot';
 
-export type ListUnorderedProps = {
-  /**
-   * Change the default rendered element for the one passed as a child, merging their props and behavior.
-   * @default false
-   */
-  asChild?: boolean;
-} & Omit<HTMLAttributes<HTMLUListElement>, 'size'>;
```

Comment:
`asChild` still needs to be there in case anyone has their own list they want to change it with.
We don't want them to use it to change to ex. a `div`, but there a still cases where someone might need to change underlying logic with another component.

Take `Link` as an example of this.